components/site/src/lib.rs

```diff
@@ -540,7 +541,24 @@ impl Site {
 
     /// Renders a single content page
     pub fn render_page(&self, page: &Page) -> Result<()> {
-        let output = page.render_html(&self.tera, &self.config, &self.library.read().unwrap())?;
+        let mut output = page.render_html(&self.tera, &self.config, &self.library.read().unwrap())?;
+
+        if self.config.minify {
+            let cfg = &Cfg {
+                minify_js: false,
+            };
+            let mut output_bytes = output.as_bytes().to_vec();
+            match truncate(&mut output_bytes, cfg) {
+                Ok(_len) => {
+                    output = std::str::from_utf8(&mut output_bytes).expect("Unable to truncate html file.").to_string();
+                },
+                Err(MinifyError{error_type, position})=> {
+                    eprintln!("Failed to truncate html at character {}:", position);
+                    eprintln!("{}", error_type.message())
+                }
+            }
+
+        }
```

Comment:
you need to have the same code everywhere we are rendering a template: section and taxonomies at least